packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -873,64 +898,67 @@ export function CreateTaskForm({
+                  <Grid item xs={8}>
                     <DateTimePicker
                       disabled={warnTime === null}
                       value={warnTime}
                       onChange={(date) => {
                         setWarnTime(date);
                       }}
                       label="Warn Time"
+                      sx={{ marginLeft: theme.spacing(1) }}
                     />
                   </Grid>
+                  <Grid container xs={3} justifyContent="flex-end" alignItems="center">
+                    <Tooltip title="Prioritized tasks will added to the front of the task execution queue">
+                      <Chip
+                        icon={<BoltIcon />}
+                        label="Prioritize"
+                        color="primary"
+                        variant={
+                          !taskRequest.priority || !parseTaskPriority(taskRequest.priority)
+                            ? 'outlined'
+                            : 'filled'
+                        }
+                        onClick={handlePriorityClick}
+                      />
+                    </Tooltip>
+                  </Grid>
                 </Grid>
                 <Divider
                   orientation="horizontal"
                   flexItem
                   style={{ marginTop: theme.spacing(2), marginBottom: theme.spacing(2) }}
                 />
                 {renderTaskDescriptionForm(taskDefinitionId)}
-                <Grid container justifyContent="center">
-                  <Button
-                    aria-label="Save as a favorite task"
-                    variant="contained"
-                    color="primary"
-                    size={isScreenHeightLessThan800 ? 'small' : 'medium'}
-                    onClick={() => {
-                      !callToUpdateFavoriteTask &&
-                        setFavoriteTaskBuffer({ ...favoriteTaskBuffer, name: '', id: '' });
-                      setOpenFavoriteDialog(true);
-                    }}
-                    style={{ marginTop: theme.spacing(2), marginBottom: theme.spacing(2) }}
-                    // FIXME: Favorite tasks are disabled for custom compose
-                    // tasks for now, as it will require a re-write of
-                    // FavoriteTask's pydantic model with better typing.
-                    disabled={taskDefinitionId === CustomComposeTaskDefinition.taskDefinitionId}
-                  >
-                    {callToUpdateFavoriteTask ? `Confirm edits` : 'Save as a favorite task'}
-                  </Button>
-                </Grid>
               </Grid>
             </Grid>
           </DialogContent>
+          <DialogActions>
+            <Button
+              aria-label="Save as a favorite task"
+              variant={callToUpdateFavoriteTask ? 'contained' : 'outlined'}
+              color="primary"
+              size={isScreenHeightLessThan800 ? 'small' : 'medium'}
```

Comment:
Avoid adding more screen size hacks.

Reply:
this was moved from https://github.com/open-rmf/rmf-web/pull/1000/files#diff-b303145e2e7e318221a93e9f20d4e8f50dab4b4aae04b4b94a8db382e7b406e4L916, I would rather clean all of them up in a separate PR